vendor/bat-native-ads/src/bat/ads/internal/account/confirmations/confirmations_state.cc

```diff
@@ -342,7 +342,7 @@ base::Value ConfirmationsState::GetFailedConfirmationsAsDictionary(
       base::DictionaryValue* user_data_dictionary = nullptr;
       if (user_data->GetAsDictionary(&user_data_dictionary)) {
         confirmation_dictionary.SetKey("user_data",
-                                       base::Value(user_data_dictionary));
```

Comment:
@moritzhaller, @tmancey, based on the comment in https://chromium.googlesource.com/chromium/src/+/1f00b01245dac4b8c0dfa70faf1a661becc3d886 the current code may not be working as intended.